src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -102,7 +102,7 @@ Here is the basic process you would use to obfuscate the sensitive data in this
             ```json
             {
               "name": "Credit Card Number",
-              "regex": "(\d{4}-\d{4}-\d{4}-\d{4})"
+              "regex": "((?:(?:4\d{3})|(?:5[1-5]\d{2})|6(?:011|5[0-9]{2}))(?:-?|\040?)(?:\d{4}(?:-?|\040?)){3}|(?:3[4,7]\d{2})(?:-?|\040?)\d{6}(?:-?|\040?)\d{5})"
```

Comment:
@lchapman4, I'm not too strong on regex, but the `...4\d{3}...` is rendering as `...4d{3}....` Is that what you expect?

Reply:
oh no, not sure why that's happening. I'll escape it and see if that helps it resolve.